src/tracker-component.js

```diff
@@ -309,7 +308,7 @@ export const Tracker = (config? : Config = {}) => {
             createNewCartId();
             return event;
         },
-        setUser: (data : UserData) => {
+        setUser: (data : { user : UserData }) => {
```

Comment:
Does this at all affect the fact that we can pass in `{ id }` or `{ user: id }` here?

Reply:
Functionally, no. But that's a good catch. Type should allow for both cases. I'll update it.